client/src/components/geneExpression/infoPanel/connect.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeEvent, useEffect, useState } from "react";
+import { AppDispatch } from "../../../reducers";
+
+function useConnect({
+  dispatch,
+  activeTab,
+}: {
+  dispatch: AppDispatch;
+  activeTab: string;
+}) {
+  const [value, setValue] = useState(1);
```

Comment:
can we use this enum? `export enum ActiveTab {` - hard coded numbers are less readable. Thank you!